liblangutil/Exceptions.h

```diff
@@ -211,11 +220,12 @@ class Error: virtual public util::Exception
 
 	static constexpr Severity errorSeverity(Type _type)
 	{
-		if (_type == Type::Info)
-			return Severity::Info;
-		if (_type == Type::Warning)
-			return Severity::Warning;
-		return Severity::Error;
+		switch (_type)
+		{
+			case Type::Info: return Severity::Info;
+			case Type::Warning: return Severity::Warning;
+			default: return Severity::Error;
```

Comment:
That's gonna be circular isn't it ? 
`isError(_type)`  calls the function `errorSeverity(_type)` itself.

Reply:
> That's gonna be circular isn't it ?

Ah, good point. Forgot that one is defined in terms of the other. It's fine then.

> Also, we've removed it from `formatErrorSeverity()` as well

Yeah, I know. I was just saying that in that particular case it wasn't very useful. But maybe it's because the code changed around it. I remember that I suggested it originally and back then it did make sense.